examples/hdf5/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2015, Oliver Daniell
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-hdf5)
+
+hunter_add_package(hdf5)
+
+find_package(ZLIB CONFIG REQUIRED)
```

Comment:
Just for your information you can use [find_dependency](http://www.cmake.org/cmake/help/v3.3/module/CMakeFindDependencyMacro.html) macro in installed `*Config.cmake` file to apply transitive search.